include/Telemetry/Telemetry.h

```diff
@@ -20,28 +84,28 @@
 /* { obj.println(); return obj; } */
 
 #define BEGIN_SEND {              \
```

Comment:
For the actual protocol, I've been considering making a significant change to this for Ethernet.  The only reason for the weird thing with `@@@@@` and `&&&&&` was a workaround for the issue that we only had one method of output, and I didn't want debugging `Serial.print(...)` statements to break telemetry.  

Since Ethernet doesn't need to include this same sort of debugging flexibility, we could do away with the opening and closing symbols, slightly improving performance.  We can change actual status/error messages, currently sent via Serial, to send as just another data field instead.  

IDK, just a thought.  But this is something we don't need to change right now, I can mess with it later.